pkg/ccl/backupccl/testdata/backup-restore/restore-tenants

```diff
@@ -18,8 +18,9 @@ SELECT crdb_internal.destroy_tenant(5);
 query-sql
```

Comment:
Might be worth it to modify this test to create a named tenant so that it ensures the names are restored for cluster restore.

Do we expect `RESTORE TENANT 5` to also restore the name of tenant 5 at the time of the backup?  If so, a test to cover that case might fit well in this file as well.  But, it may also be fair to simply open an issue to track the behaviour of that.